Sentry.Xamarin.Forms/SentryXamarinFormsIntegration.cs

```diff
@@ -43,9 +43,17 @@ public void Register(IHub hub, SentryOptions options)
             _hub = hub;
 
             options.AddEventProcessor(new XamarinFormsEventProcessor(options));
-#if !NETSTANDARD
-            options.AddEventProcessor(new NativeEventProcessor(options));
-#endif
+
+            var nativeEventProcessor = new NativeEventProcessor(options);
+            if (nativeEventProcessor.Implemented)
```

Comment:
yeah this shouldn't go like this.

Maybe you need a `#if NATIVE_PROCESSOR` and conditionally compile the whole thing

Reply:
if added